### ISSUE TITLE
Regularization with torch-pruning v1.0.0

### ISSUE THREAD

**Issue by Serjio42** [VainF/Torch-Pruning#128]
Hi again.
I want to try regularization of pruning groups (aka _sparse training_ / _sparse learning_, as I understood correctly).
I wonder whether it will work correctly with pip available library version 1.0.0? When I go to `1.0` branch I do not see Sparse Training section in ReadMe. Maybe it's worth to update pip package?
What is common practice to take advantage from sparse training in user training loop? Using `GroupNormPruner` (with `pruner.regularize(model)`) and `GroupNormImportance` as I understand form ReadMe...

**Comment by VainF** [comment]
Yes, sparse training should work with v1.0. It is used to zeroize some unimportant channels so that they can be safely removed without too much performance degradation.

**Comment by Serjio42** [comment]
Great, thanks! I'll try it.

**Comment by VainF** [comment]
PyPI package updated! Thank you~

**Comment by Serjio42** [comment]
Thank you for your efforts! Keep learning/coding.
